src/system/Toolbar/Logo.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+const VIP_LOGO = 'vip-logo-component';
+
+type LogoProp = {
+	className?: string;
+	href?: string;
```

Comment:
looks required
```suggestion
	href: string;
```

Reply:
@chriszarate I am still thinking about this, I think we need a `asChild` there to allow Next.js customization

Reply:
Seems like you should just export the logo and let the implementor wrap in a Next.js Link?

Reply:
@chriszarate Two reasons I prefer using the way it is now:

- Wrapping with a link won't get styles we need to the Toolbar
- People can extend and do what ever they want with this link, wrong padding, etc
- `as` is built-in and flexible to allow you to use Next.js links too